easybuild/framework/easyconfig/tools.py

```diff
@@ -465,3 +467,61 @@ def review_pr(pr, colored=True, branch='develop'):
             lines.extend(['', "(no related easyconfigs found for %s)\n" % os.path.basename(ec['spec'])])
 
     return '\n'.join(lines)
+
+
+def dump_env_script(easyconfigs):
+    """
+    Dump source scripts that set up build environment for specified easyconfigs."""
+    ecs_and_script_paths = []
+    for easyconfig in easyconfigs:
+        script_path = '%s.env' % os.path.splitext(os.path.basename(easyconfig['spec']))[0]
```

Comment:
not sure about this...

it's a script, so `.txt` doesn't make sense to me, but I also don't want to use `.sh` to make it clear it should be *sourced*

is there a standard for something like this?